packages/client/src/common/components/NavBar/NavBar.style.tsx

```diff
@@ -0,0 +1,11 @@
+import { Box, Toolbar } from '@mui/material';
```

Comment:
It does not return any **tsx** element, you can rename it with the **.ts** extension